antora-playbook.yml

```diff
@@ -1,14 +1,23 @@
 site:
   title: Apache Camel
   url: https://camel.apache.org
-  start_page: manual::index.adoc
+  start_page: latest@manual::index.adoc
 
 content:
   sources:
-    - url: git@github.com:apache/camel.git
+    - url: https://github.com/djencks/camel.git
```

Comment:
We need to change the repository to have a full test.

Reply:
OOPS!  I pushed a fix that works OK for me locally.

BTW when I run mvn clean install and then git status there are an enormous number of files deleted from static.  Is this expected?

Reply:
No, this shouldn't happen. I can reproduce it locally and I don't know why. The schemas must be there as all the static content.

Reply:
So please don't push that part and otherwise open an issue for this, so we can double check why

Reply:
fixed here https://github.com/apache/camel-website/pull/240